src/Text/Pandoc/Readers/Markdown.hs

```diff
@@ -1737,15 +1737,15 @@ linkTitle :: PandocMonad m => MarkdownParser m Text
 linkTitle = quotedTitle '"' <|> quotedTitle '\''
 
 -- Github wiki style link, with optional title
+-- syntax documented under https://help.github.com/en/github/building-a-strong-community/editing-wiki-content
 githubWikiLink :: PandocMonad m => MarkdownParser m (F Inlines)
 githubWikiLink = try $ do
-  guardEnabled Ext_markdown_github_wikilink
-  string "[["
-  skipSpaces
-  let allUntil = manyTillChar (satisfy $ const True)
-  part1 <- allUntil (oneOf "|]")
-  part2 <- (char ']' >> return part1) <|> allUntil (string "]]")
-  returnF $ B.link part1 "" (B.text $ escapeURI $ trimr part2)
+  guardEnabled Ext_wikilink
+  try $ string "[["
```

Comment:
You don't need this `try` or the one on l 1747 because you've got a `try` over the whole `githubWikiLink` parser (which maybe should be named `wikiLinkParser`?)  Having the unnecessary `try`s just hurts performance.